apps/webapp/src/components/RootLayout/index.tsx

```
@@ -63,87 +63,71 @@
+                <div className='flex gap-2 items-center'>
+                  <TeamSwitcher className='w-40 ml-3' projects={projectsData? projectsData : []}/>
+                  <div 
+                    className="cursor-pointer" 
+                    onClick={handleCopyRight}
+                  >
+                    <TooltipProvider delayDuration={0}>
+                      <Tooltip>
+                        <TooltipTrigger asChild>
+                          <Button size="sm" variant="outline">
+                            {copiesProjectID ? (
+                              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" width="15" height="15" fill="#ffffff">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)